src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -3006,7 +3137,15 @@ public override bool Execute()
                 if (connected)
                 {
                     // Client is connected to the RAR node, we can execute RAR task remotely
-                    // return client.Execute(); // TODO: Let it do something.
+                    MSBuildEventSource.Log.RARaaSStart();
+                    ResolveAssemblyReferenceResult result = client.Execute(ResolveAssemblyReferenceInput);
+                    MSBuildEventSource.Log.RARaaSStop();
+                    ResolveAssemblyReferenceOutput = result.Output;
+                    LogEvents(result.BuildWarningEvents);
```

Comment:
Can you please explain why logging is split into warnings / messages / error / etc. when `LogEvents` has a per-item switch? Does this mean that all warnings are reported before all messages and those in turn before all errors?